### ISSUE TITLE
[BUG] Wallet not syncing, Response highly appreciated, its been going on for weeks

### ISSUE THREAD

**Issue by felebowski** [Chia-Network/chia-blockchain#5660]
**Describe the bug**
Every now and then my wallet goes out of sync and farming stops. It says Connected and Syncing, but it does not Sync. Plotting however works, but i am not able to farm. I have the error message in my developer tools DevTools failed to load SourceMap: Could not parse content for file:///C:/Users/felix/AppData/Local/chia-blockchain/app-1.1.6/resources/app.asar/build/renderer/js/80.js.map: Unexpected end of JSON input.
Even uninstalling, deleting the folders and installing it again does not help. 

**To Reproduce**

1. Go to Full Node
2. Look at Status
3. See its not syncing. 
4. Go to developer tools, check table 

**Expected behavior**
I expect the wallet to sync and farm
**Screenshots**

**Desktop (please complete the following information):**

- OS: Win10 Home 
- OS 2004
- CPU: Intel Core i7-10700F

**Comment by rvp8762446** [comment]
Hi,

Syncing issues seem to have been fixed for me with release 1.1.6 (so far, just installed it). I suggest that if both your full node and wallet are not syncing, try to uninstall everything (again). 
- Make sure you have the 24 word key u used, so you can use to import it again later. 
- If you do have you node synced to a certain height, you could try to make a backup of ' C:\Users\felix\.chia\mainnet\db\blockchain_v1_mainnet.sqlite ' This can save you same time syncing again. However, if you are not synced at all (Peik height = 0 or very low compared to where the blockchain is at now), don't bother with backing that up. 
- Same thing applies regarding your wallet: C:\Users\felix\.chia\mainnet\wallet\db\blockchain_wallet_v1_mainnet_*.sqlite (* is a unique number), you could either try to safe time to try and continue at height it stopped before, or start syncing all over. 

To uninstall:
Make sure you stopped all the proceses completely:
- stop chia by closing the window
- reboot pc (to make sure all background processes are completely stopped)
- uninstall ' chia blockchain'  through programs and features in windows control panel
Check for leftover folders and delete (or rename) them: 
- C:\Users\felix\.chia
- C:\Users\felix\AppData\Local\chia-blockchain
- C:\Users\felix\AppData\Roaming\Chia Blockchain
- Reboot PC again, just to be sure

To install: follow Wiki installation instructions
- Although Upnp can be used, better to check if port 8444 is open. If not open/port forward 8444 (see: https://github.com/Chia-Network/chia-blockchain/wiki/Resolving-Sync-Issues---Port-8444)
- Use latest release (1.1.6 as I am writing)
- Use the key you used earlier so you are using the same wallet and can use the plots you have created earlier to farm
- Look at full node: status should go from 'not synced'  to syncing pretty fast now. Scroll down and you should see connections appearing automatically (there should be no need to manually connect to other peers)
- Go to wallet. Choose to either use backup file (for couloured coins) or skip. In my experience, the wallet will not start syncing right away. It can take up to 15 minutes or so untill it starts syncing
- If Full Node is completely synced and you have added you plot directories containing your finished plots, farming should start automatically. As long as full node is not synced, farming will not occur. However, the wallet does not need to be fully synced to be farming, but of course you will not be able to see any balance you may have, untill the wallet is synced to a certain height.

If syncing still does not work, try to analyse further: start to learn how to use the debug logs (change log level to INFO) and how to interpret them (I am still learning myself). It could be network related, firewalll related etc, hard to tell.

I hope this helps you further.

**Comment by rvp8762446** [comment]
However... As I was typing my reply, having installed 1.1.6, on my windows install I am having (only wallet, node is fine) syncing issues again: 
- Stopped syncing @202987
- Found following in log: 2021-05-21T08:38:59.304 wallet chia.wallet.wallet_node    : ERROR    Loop exception in sync Was not able to add blocks 203007-203039. Traceback (most recent call last):

-I stopped all Chia processes, reopened.  Now the GUI stated that is was stuck at 203007 suddenly (which makes more sense comparing to error I found in log..)
- waited a few minutes, and at some point it started syncing again fortunately. So even release 1.1.6 seems to fix a lot of issues/improve things, it still seems to require constant monitoring.
